src/plugin.js

```diff
@@ -206,8 +206,31 @@ const contribAdsPlugin = function(options) {
       return !videojs.browser.IS_IOS &&
              !videojs.browser.IS_ANDROID &&
              somePlayer.duration() === Infinity;
-    }
+    },
 
+    // Return true if the ad should NOT autoplay on iPhone on iOS 10 and above.
+    // This is only true if the player does not have autoplay, playsinline
+    // and muted attributes or player settings. Otherwise, we should attempt
+    // to autoplay the ad.  Also pauses the player if it has started playing and
+    // removes the autoplay attribute or player setting.
+    cancelAutoplayAdOnIOS(somePlayer) {
+      if (videojs.IS_IPHONE && player.el_.hasAttribute('playsinline') &&
+        (player.autoplay() || player.el_.hasAttribute('autoplay')) &&
+        (!player.muted() || !player.el_.hasAttribute('muted'))) {
+
+        if (!player.paused()) {
+          player.pause();
```

Comment:
This is a side effect of this boolean check, we should move this out into an else block.